boa/src/builtins/symbol/mod.rs

```diff
@@ -91,6 +91,11 @@ impl BuiltIn for Symbol {
         )
         .callable(true)
         .constructable(false)
+        .property(
+            symbol_to_string_tag,
+            "Symbol",
```

Comment:
I would use `Self::NAME` here.

Reply:
Yes I will change that 
Thanks for the review